cms/envs/common.py

```diff
@@ -1085,7 +1085,8 @@
 }
 
 DEFAULT_AUTO_FIELD = 'django.db.models.AutoField'
-DEFAULT_HASHING_ALGORITHM = 'sha1'
+
+DEFAULT_HASHING_ALGORITHM = 'sha256'
```

Comment:
For django42 compatiblity.

Reply:
if we merge this with django32 it will logout users but when we finally upgrade to django42 this variable can be remove and it will automatically use `256`